packages/web3-eth/test/integration/web3_eth/send_transaction.test.ts

```diff
@@ -0,0 +1,121 @@
+			const transaction: Transaction = {
+				from: accounts[0],
+				data: greeterContractDeploymentData,
+				gas: '0x740b8',
+			};
+			const response = await web3Eth.sendTransaction(transaction);
+			expect(response.status).toBe('0x1');
+			expect(response.contractAddress).toBeDefined();
+
+			greeterContractAddress = response.contractAddress as string;
+		});
+
+		it('should update greet in contract', async () => {
+			const contractFunctionCall =
+				'0xa4136862000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000023432000000000000000000000000000000000000000000000000000000000000';
+			const transaction: Transaction = {
+				from: accounts[0],
+				to: greeterContractAddress,
+				data: contractFunctionCall,
+			};
+			const response = await web3Eth.sendTransaction(transaction);
+			expect(response.status).toBe('0x1');
+		});
+	});
+
+	describe('Transaction Types', () => {
+		it('should send a successful type 0x0 transaction', async () => {
+			const transaction: Transaction = {
+				from: accounts[0],
+				to: '0x0000000000000000000000000000000000000000',
+				value: '0x1',
+				type: '0x0',
+			};
+			const response = await web3Eth.sendTransaction(transaction);
+			expect(response.type).toBe('0x0');
+			expect(response.status).toBe('0x1');
+		});
+
+		it('should send a successful type 0x1 transaction', async () => {
+			const transaction: Transaction = {
+				from: accounts[0],
```

Comment:
In this test could you use local wallet account `address` instead of index, this will cover one more scenario in e2e, and match with mined tx `from` field in receipt

Reply:
Addressed by [this](https://github.com/ChainSafe/web3.js/pull/5056/commits/2de671fbcab7c82e5de45cf31f039af33bec804f) commit